tests/test_hooks.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+from shutil import copy2
+
+import pytest
+
+from hookman.hooks import HookMan, HooksSpecs
+
+
+
+@pytest.fixture()
+def plugin_specs():
+    """
+    This plugin specs has the same interface as the simple_plugin
+    """
+
+    def friction_factor(v1: 'int', v2: 'int') -> 'int':
+        """
+        Docs for Friction Factor
+        """
+
+    return HooksSpecs(project_name='acme', version='4', pyd_name='_test_hooks', hooks=[friction_factor])
+
+
+def test_find_plugin(datadir, libs_path, plugin_specs):
+
+    HookMan(specs=plugin_specs, plugin_dirs=[datadir])
+
+    with pytest.raises(FileNotFoundError, match="The given path doesn't have a .yaml file"):
+        HookMan(specs=plugin_specs, plugin_dirs=[Path('NON_EXISTING_PATH')])
+
+
+def test_check_hook_arguments():
+
+    def test_method_without_arguments() -> 'float':

```

Comment:
I suggest to write 3 separate tests, instead of bundling them together here.

Reply:
- [x] separate in 3 tests